ui/src/components/TaskRunsCard/index.jsx

```diff
@@ -94,6 +94,11 @@ const DOTS_VARIANT_LIMIT = 5;
       display: 'inline-flex',
       flexBasis: '50%',
     },
+    articleName: {
```

Comment:
Did you mean `artifactName`? :)

Reply:
@helfi92  Yeah .sorry for the mis-spelling. Let me fix it now :) . Really sorry

Reply:
@helfi92  I fixed it .